lib/beaker-puppet/install_utils/foss_utils.rb

```diff
@@ -438,11 +437,7 @@ def configure_puppet_on(hosts, opts = {})
         # @api private
         def install_puppet_from_rpm_on( hosts, opts )
           block_on hosts do |host|
-            if host[:type] == 'aio'
```

Comment:
Ah yes, I see where you're coming from here. That makes sense. Yes, I'll add in a work around for this. Do you think it's safe to change the default repo that is enabled, or would that be too much of a breaking change?